packages/titanium-docgen/index.js

```diff
@@ -883,6 +917,14 @@ if (basePaths.length === 0) {
 	process.exit(1);
 }
 
+const { version } = require(pathMod.resolve(basePaths[0], '..', 'package.json'));
```

Comment:
Just to be sure, check with `fs.existsSync` before doing the require here. I'm not 100% sure there will always be a `package.json` present in all projects we use the docgen on. The `require` for the version was previously only run when the format was either `typescript` or `changes` and we did definitely only run those formats for the SDK itself, so that wasn't an issue there.

Reply:
done